packages/yourloops/components/patient/patient-row.tsx

```diff
@@ -171,130 +128,137 @@ function PatientRow(props: PatientElementProps): JSX.Element {
-        {isUserHcp && nonDataTransmissionActive &&
-          <AnnouncementIcon titleAccess="no-data-alert-icon" className={classes.alertIcon} />}
-      </StyledTableCell>
-      <StyledTableCell id={`${rowId}-ldu`} className={classes.typography}>
-        {lastUpload}
-      </StyledTableCell>
-      <StyledTableCell id={`${rowId}-messages`}>
-        {patientIsMonitored && patient.metadata.unreadMessagesSent > 0 &&
-          <EmailIcon titleAccess="unread-messages-icon" className={classes.coloredIcon} />
+
+        <StyledTableCell
+          id={`${userId}-patient-full-name`}
+          className={patientFullNameClasses}
+        >
+          <Tooltip title={tooltipText}>
+            <Typography
+              id={`${userId}-patient-full-name-value`}
+              data-testid="patient-full-name-value"
+              className={classes.typography}
+            >
+              {patientFullName}
+            </Typography>
+          </Tooltip>
+        </StyledTableCell>
+
+        <StyledTableCell className={classes.typography}>{patientSystem}</StyledTableCell>
+        {isUserHcp &&
+          <StyledTableCell
+            className={`${classes.typography} ${patientListCommonClasses.mediumCell} ${classes.remoteMonitoringCell}`}>
+            {patientRemoteMonitoring}
+          </StyledTableCell>
         }
-      </StyledTableCell>
-    </StyledTableRow>
+
+        <StyledTableCell className={timeSpentAwayFromTargetRateClasses}>
+          {`${Math.round(patient.metadata.alarm.timeSpentAwayFromTargetRate * 10) / 10}%`}
+          {isUserHcp && timeSpentAwayFromTargetActive &&
+            <AnnouncementIcon titleAccess="time-away-alert-icon" className={classes.alertIcon} />}
+        </StyledTableCell>
+
+        <StyledTableCell className={frequencyOfSevereHypoglycemiaRateClasses}>
+          {`${Math.round(patient.metadata.alarm.frequencyOfSevereHypoglycemiaRate * 10) / 10}%`}
+          {isUserHcp && frequencyOfSevereHypoglycemiaActive &&
+            <AnnouncementIcon titleAccess="severe-hypo-alert-icon" className={classes.alertIcon} />}
+        </StyledTableCell>
+
+        <StyledTableCell className={dataNotTransferredRateClasses}>
+          {`${Math.round(patient.metadata.alarm.nonDataTransmissionRate * 10) / 10}%`}
+          {isUserHcp && nonDataTransmissionActive &&
+            <AnnouncementIcon titleAccess="no-data-alert-icon" className={classes.alertIcon} />}
+        </StyledTableCell>
+
+        <StyledTableCell className={classes.typography}>
+          {lastUpload}
+        </StyledTableCell>
+
+        <StyledTableCell>
+          {patientIsMonitored && patient.metadata.unreadMessagesSent > 0 &&
+            <EmailIcon titleAccess="unread-messages-icon" className={classes.coloredIcon} />
+          }
+        </StyledTableCell>
+
+        <StyledTableCell>
+          {isUserHcp &&
+            <IconActionButton
+              ariaLabel="remove-patient-button"
```

Comment:
ariaLabel should be a translated string such as 'remove patient" (I know this has not been well respected in the app).
I also think that it would be a good idea to have a tooltip on this button (also with the label 'remove patient'). What do you think?